packages/dashboard/src/root/navbar.js

```diff
@@ -166,7 +211,11 @@ class DashboardNavbar extends LitElement {
         <div class="tabs">
           <vaadin-tabs .selected=${this.selectedTabIndex} @selected-changed=${this.#onTabChange}>
             ${tabs.map(({ label }, index) => html`
-            <vaadin-tab value=${index}>${label}</vaadin-tab>
+            <vaadin-tab value=${index}>${label} 
+              <vaadin-button class="delete-tab-button" theme="icon tertiary" aria-label="Delete Tab" deletes=${label} @click=${()=>this.deleteTab(label)}
+              ?disabled=${!this.dashboard.isDrawerOpened()}>
+              <vaadin-icon icon="vaadin:close"></vaadin-icon>
+            </vaadin-tab>
```

Comment:
This looks good. Could the icon be made a little smaller? I think it looks better when it's less prominent:

<img width="389" alt="tabs-with-x" src="https://user-images.githubusercontent.com/6786620/188548578-c95e77db-e3d8-4af5-b752-ed411f314a9b.png">

Could you also add `cursor: pointer` to the icon styles?